### PR TITLE
Attempt to boot the gleam binary after building release

### PR DESCRIPTION
**Pull request by thepratt**
Aim is to round up and tick off some of the CI tagged tickets, with #769 being the first up.

TODO:
- [x] Validate 7zip commands w/ a windows prompt

### CODE REVIEW

**Comment by lpil** [comment]
Oh brilliant, thank you!

RE release creation, I'm happy to have the draft be created even if it fails. I can delete the draft release no problem, and it's less work than fixing the build which I would have to do anyway.

**Comment by thepratt** [comment]
No problem! Looking to help with the small stuff before diving into bigger projects.

Good to know RE releases, just need to get myself a windows cmd to play around with then - or if anyone else can validate the 7zip commands.

**Comment by lpil** [comment]
Is this ready to review? There's a TODO in the description but the code looks good to me. I have not run it though, have you?

**Comment by thepratt** [comment]
Shouldn't have moved it from draft, will wait on #1420 b/c there will be conflicts.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -112,6 +123,12 @@ jobs:
           tar -czvf "$ARCHIVE" "gleam"
           echo "ASSET=$ARCHIVE" >> $GITHUB_ENV
 
+      - name: Ensure binary sucessfully boots
```
Comment (JohnDoneth) [inline .github/workflows/release.yaml:126]:
```suggestion
      - name: Ensure binary successfully boots
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -76,6 +76,17 @@ jobs:
             echo "ASSET=$ARCHIVE" >> $GITHUB_ENV
           fi
 
+      - name: Ensure binary sucessfully boots
```
Comment (JohnDoneth) [inline .github/workflows/release.yaml:79]:
```suggestion
      - name: Ensure binary successfully boots
```

**Review by JohnDoneth — COMMENTED** [review]
Couple typos

File: .github/workflows/release.yaml (hunk 1 above)
Comment (thepratt) [reply .github/workflows/release.yaml]: I'll learn to spell one of these days

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**